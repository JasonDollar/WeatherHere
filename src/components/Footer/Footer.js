import React from 'react'
import classes from './Footer.module.scss'

const Footer = () => (
  <div className={classes.Footer}>
    <a href="https://darksky.net/poweredby/" className={classes.link}>Powered by DarkSky API &rarr;</a>
  </div>
)

export default Footer
