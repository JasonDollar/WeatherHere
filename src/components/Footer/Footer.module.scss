.Footer {
  
  // background-color: var(--color-back);
  margin: 1rem auto;
  font-size: 2rem;
}

.link:link,
.link:visited {
  color: inherit;
  text-decoration: none;
  cursor: pointer;
  padding: 1rem 1rem;
  border-bottom: 2px solid transparent;
  transition: .15s all;
}

.link:hover,
.link:active {
  border-bottom: 2px solid var(--color-primary-offset);
}