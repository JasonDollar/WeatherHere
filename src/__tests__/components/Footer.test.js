import React from 'react'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
configure({ adapter: new Adapter() })
import Footer from '../../components/Footer/Footer'

test ('render footer', () => {
  const wrapper= (<Footer />)
  expect(wrapper).toMatchSnapshot()
})