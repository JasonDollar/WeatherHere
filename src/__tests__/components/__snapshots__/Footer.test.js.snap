// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer to match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <a
        className="link"
        href="https://darksky.net/poweredby/"
      >
        Powered by DarkSky API →
      </a>,
      "className": "Footer",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Powered by DarkSky API →",
        "className": "link",
        "href": "https://darksky.net/poweredby/",
      },
      "ref": null,
      "rendered": "Powered by DarkSky API →",
      "type": "a",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <a
          className="link"
          href="https://darksky.net/poweredby/"
        >
          Powered by DarkSky API →
        </a>,
        "className": "Footer",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Powered by DarkSky API →",
          "className": "link",
          "href": "https://darksky.net/poweredby/",
        },
        "ref": null,
        "rendered": "Powered by DarkSky API →",
        "type": "a",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
